cmd/delete_temp_users.go

```diff
@@ -44,5 +58,8 @@ func init() { //nolint:gochecknoinits,gocyclo
 		},
 	}
 
+	deleteTempUsersCmd.Flags().DurationVar(&delay, "delay", 60*24*time.Hour,
+		"delay between expiration and deletion of temporary users")
```

Comment:
it should mention the unit!

Reply:
> it should mention the unit!

The unit is specified in the command line. It looks like "1440h" or "30m".